kotlintest-core/src/commonMain/kotlin/io/kotlintest/Spec.kt

```diff
@@ -74,7 +74,7 @@ interface Spec : TestListener {
 
    fun description(): Description = Description.fromSpecClass(this::class)
 
-   val assertionMode: AssertionMode? get() = null
```

Comment:
I believe you added this recently, so there would be no problems to refactor this, right? It didn't go into a release version

Reply:
Correct. I only added it for 4.0 anyway.